Public/Src/Sandbox/MacOs/Sandbox/Src/BuildXLSandboxShared.hpp

```diff
@@ -259,10 +259,24 @@ inline bool HasAllFlags(const T source, const T bitMask)
 
 #pragma mark Macros and defines
 
-#define BXL_CLASS(name) com_microsoft_buildxl_ ## name
-#define BuildXLSandbox BXL_CLASS(Sandbox)
-#define kBuildXLBundleIdentifier "com.microsoft.buildxl.sandbox"
-#define kBuildXLSandboxClassName "com_microsoft_buildxl_Sandbox" // this string must match the value of BuildXLSandbox
+#ifndef BXL_BUNDLE_IDENTIFIER
```

Comment:
`BUILDXL_` for consistency with windows?

Reply:
will rename